### DISCUSSION TITLE
Access from the web

### DISCUSSION THREAD

**Discussion by kgreene1983** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#295] [Q&A]
Hello.

So I'm tying to work out how I can access my spa from the WWW.

So I have an ASUS router and have a ddns address. 

I tried to set up some port forwarding but no look ie HTTP with port 80 local IP of spa. 

Can anybody point in the correct direction

**Comment by chunkysteveo** [comment]
Please don't do this... you would be giving anyone access to a +2000W heating element and blower fan with minimal security. A more suitable solution would be first to setup a VPN on your network, your router may offer this, and then conrol the Spa when logged in via the VPN. 

The project has basic webpage security, but it's not designed to be be secure or hack proof.

That being said - you would open a different port on the internet side, e.g. :8079 and route external inbound traffic on that port internally to your Spa's IP address via internal port 80. Port 80 on your router via the internet will most likely stop at your router's web interface.

Steve

**Reply by paulknox** [reply]
I'm trying to do the same... I have a synology nas which I believe can be setup as a vpn server so I think i might try setting this up and then trying to port forward using the synology...

**Reply by chunkysteveo** [reply]
That's how my VPN is setup - I use OpenVPN on my Synology and can tunnel into my home network from there.... There's a lot of guides to set this up online - Windows OpenVPN and Android Apps work well. Best of luck!

**Reply by kgreene1983** [reply]
Ahhh I have a synology NAS so will look at that.

**Reply by paulknox** [reply]
Do I need to port forward from the synology vpn to the WiFi ip and port when I set it up?

**Reply by chunkysteveo** [reply]
No - when you are successfully logged into your VPN on your device from outside your network - you are essentially "on you LAN", so you can (should be able to) enter the IP address of the Spa in a browser and it should come up. Same with anything else on your local network - when you are VPN'd in, you are ON your local LAN.

**Reply by paulknox** [reply]
Just worked that out lol!! Perfect and now up and running through the vpn with all port forwarding removed from my mesh deco and adsl modem! Thanks for point me in the right direction!

**Reply by chunkysteveo** [reply]
No worries. As good and handy and safe (if done properly) opening ports can be - locking your local ADSL and ONLY having a VPN is a much safer way to manage your local LAN while out and about. Gives hackers less options to try to sneak in! 

Same safe option if you have a 3D printer and use Octoprint - you wouldn't want external people controlling your 3D printer and hotend - but with a VPN you can log in and monitor the prints via the webcam as if you are sat on your own network.

The Android app for OpenVPN is really good too!

**Reply by mulroux95** [reply]
hello, I would like to redirect first but the spa information but on the other hand the display of the web page is present
![Screenshot_2022-09-05-18-36-04-658_com android chrome](https://user-images.githubusercontent.com/112511099/188699889-2ab5f104-884f-4775-ba4d-77bba5bc8d39.jpg)
Any ideas?
Thanks you

**Reply by visualapproach** [reply]
@mulroux95 I don't understand your question. Maybe others don't either, considering the lack of replies.
